src/styles/stars.scss

```diff
@@ -0,0 +1,22 @@
+.starts {
```

Comment:
There is a typo in the class name. It should be `.stars` instead of `.starts` to match the expected class name in the HTML.